ui/src/app/utils/index.tsx

```diff
@@ -585,8 +585,13 @@ export const cond = <T extends unknown>(
   }
 };
 
+// normalize username (email) to remove the domain if it exists
 export const usernameWithoutDomain = (username: string) => {
```

Comment:
Before: only returns correct output if the domain is present
Now: returns correct output whether or not the domain is present